models/view/src/index.ts

```diff
@@ -215,10 +215,6 @@ export function createModel (builder: Builder): void {
     },
     view.action.Move
   )
-
-  builder.mixin(core.class.Space, core.class.Class, view.mixin.AttributePresenter, {
-    presenter: view.component.ObjectPresenter
```

Comment:
May be we just need another simple presenter? Did you check application in recruit, it can be broken

Reply:
Updated to `SpacePresenter`